VERSION

```diff
@@ -1 +1 @@
-2.5.99
+2.5.99-dev1
```

Comment:
I think we don't update the version file: For example, https://github.com/nrfconnect/sdk-nrf/pull/12078/files.

Reply:
Discussed offline. please disregard this comment.